faust-mode.el

```diff
@@ -171,7 +171,7 @@
 (defvar faust-regexp-keywords-ui (regexp-opt faust-keywords-ui 'words))
 (defvar faust-regexp-keywords-lib (regexp-opt faust-keywords-lib 'words))
 
-(defconst faust-regexp-faust-operator (rx (any ",:*-+><")))
+(defconst faust-regexp-faust-operator (rx (any "',:*-+><")))
```

Comment:
Tiny suggestion: maybe put the single quote elsewhere in the string, so it's not next to the double quote?
I think that would improve readability.

Reply:
I'm not even sure whether it should be in there at all.

The problem with `faust-regexp-faust-operator` is that it's really an incomplete and ad-hoc solution. As a result, fontification of operators is rather sketchy and inconsistent right now, e.g., in `<=` only the `<` is fontified but not the `=`. The arithmetic ops `+-*` are in there but not `/` and `%`. Some BDA ops are completely missing, such as `@` (even @rukano's original version already had that).

This deserves more scrutiny, I'll open a separate issue for it once this has been merged.